packages/jmix-front-generator/src/generators/react-typescript/app/template/i18n-message-packs/fr.json

```diff
@@ -1,4 +1,6 @@
 {
+    "addons.Addons": "Addons",
```

Comment:
This doesn't look like French to me. You can use `addons.Addons [TRANSLATION NOT AVAILABLE]` placeholder instead.